test/dash/dash_parser_manifest_unit.js

```diff
@@ -367,28 +360,76 @@ describe('DashParser Manifest', () => {
 
         /** @type {shaka.extern.Manifest} */
         const manifest = await parser.start('dummy://foo', playerInterface);
-        // First Representation should be dropped.
         const stream1 = manifest.variants[0].video;
         const stream2 = manifest.variants[1].video;
-        const stream3 = manifest.variants[2].video;
 
         const expectedClosedCaptions1 = new Map(
             [['CC1', shaka.util.LanguageUtils.normalize('eng')],
               ['CC3', shaka.util.LanguageUtils.normalize('swe')]]
         );
 
         const expectedClosedCaptions2 = new Map(
-            [['CC1', shaka.util.LanguageUtils.normalize('fr')],
-              ['CC3', shaka.util.LanguageUtils.normalize('zh')]]
-        );
-
-        const expectedClosedCaptions3 = new Map(
-            [['CC1', shaka.util.LanguageUtils.normalize('ru')],
-              ['CC3', shaka.util.LanguageUtils.normalize('hu')]]
+            [['svc1', shaka.util.LanguageUtils.normalize('bos')],
+              ['svc3', shaka.util.LanguageUtils.normalize('cze')]]
         );
         expect(stream1.closedCaptions).toEqual(expectedClosedCaptions1);
         expect(stream2.closedCaptions).toEqual(expectedClosedCaptions2);
-        expect(stream3.closedCaptions).toEqual(expectedClosedCaptions3);
+      });
+
+  it('correctly parses CEA-708 caption tags with service numbers and languages',
+      async () => {
+        const source = [
+          '<MPD minBufferTime="PT75S">',
+          '  <Period id="1" duration="PT30S">',
+          '    <AdaptationSet mimeType="video/mp4" lang="en" group="1">',
+          '      <Accessibility schemeIdUri="urn:scte:dash:cc:cea-708:2015"',
+          '         value="1=lang:eng;3=lang:swe,er"/>',
```

Comment:
Isn't this redundant with the test case above?  I can't seem to spot the difference between this 708 signaling and the 708 signaling in the previous test.

Reply:
Yes, you're right. The purpose of this test was to specifically parse only CEA-708, whereas the purpose of the previous one was to parse CEA-608 and CEA-708 when they are seen together in the same manifest.

Reading the test descriptions I can absolutely see why this is confusing - I've renamed the previous test to `correctly parses mixed captions with channels, services, and languages`